perfdash/Makefile

```diff
@@ -1,7 +1,7 @@
 all: push
 
 # See pod.yaml for the version currently running-- bump this ahead before rebuilding!
-TAG = 1.0
+TAG = 1.01
```

Comment:
Why 1.01 instead of 1.1?

Reply:
For some reason I thought that we are always using two digits for minor version. I will change it to 1.1 if I assumed incorrectly.

Reply:
It's usually just a serially increasing number for the minor version. (btw - naming it as 1.01 wouldn't let us have more than 99 minor versions in any decent way :)